wesnoth-ei.po

```diff
@@ -11601,169 +11577,156 @@ msgstr ""
 "exiled from Wesnoth. Be grateful for Our mercy."
 msgstr ""
-"Žoldnéři, <i>nekromante</i>, věz, že Wesnoth nikdy nebude tolerovat tvůj "
-"druh. Odevzdáš tu prokletou hůl ke zničení, pod trestem smrti se zřekneš "
-"provozování veškeré magie a tímto jsi vypovězen z Wesnothu. Buď vděčný za "
-"naši milost."
+"Žoldnéři, <i>nekromante</i>, věz, že Wesnoth nikdy nebude tolerovat takové "
+"spojení. Odevzdáš tu prokletou hůl, aby byla zničena, zřekneš se pod trestem "
+"smrti praktikování veškeré magie a budeš tímto  vypovězen z Wesnothu. Buď "
+"vděčen za Naše milosrdenství."
 
 #. [message]: speaker=Addogin
 #: data/campaigns/Eastern_Invasion/scenarios/18_Epilogue.cfg:391
-#, fuzzy
 msgid ""
 "Yeah, figures. How did I let myself get caught up in all this anyways? "
 "Suppose I can always head north to those farmlands we passed by..."
 msgstr ""
-"Jo, čísla. Jak jsem se do toho mohl nechat zatáhnout? Asi se vždycky můžu "
-"vydat na sever do těch zemědělských oblastí, kolem kterých jsme projížděli..."
+"Ano, chápu. Jak jsem se jen do toho všeho mohl nechat zatáhnout? "
+"Předpokládám, že vždycky můžu zamířit na sever do těch zemědělských oblastí, "
+"kolem kterých jsme projížděli..."
 
 #. [message]: speaker=Konrad
 #: data/campaigns/Eastern_Invasion/scenarios/18_Epilogue.cfg:400
-#, fuzzy
 msgid ""
 "Addogin, mercenary captain from the Estmarks. I have heard the story of how "
 "you entered Gweddry’s service against your will, but I have also heard that "
 "you honored your word to see this invasion through to its end. The palace "
 "could use a warrior of such caliber. We would offer you a position in the "
 "Royal Guard."
 msgstr ""
-"Addogin, žoldnéřský kapitán z Estmarky. Slyšel jsem příběh o tom, jak jsi "
-"proti své vůli vstoupil do Gweddryho služeb, ale také jsem slyšel, že jsi "
-"dodržel své slovo a dotáhl tuto invazi do konce. Palác by potřeboval "
-"válečníka takového kalibru. Nabízíme ti místo v královské gardě."
+"Adogine, žoldnéřský kapitáne z Východního pohraničí. Slyšeli jsme příběh o "
+"tom, jak jsi proti své vůli vstoupil do Gweddryho služeb. Ale také jsme "
+"slyšeli, že jsi dodržel své slovo a setrval až do konce této invaze. Palác "
+"by potřeboval válečníka takového kalibru. Nabízíme ti místo v královské "
+"gardě."
 
 #. [message]: speaker=Addogin
 #: data/campaigns/Eastern_Invasion/scenarios/18_Epilogue.cfg:404
-#, fuzzy
 msgid ""
 "Sir, uh, milord, uh, King sir, I’m honored, really am, but I just wanna go "
 "home. If it’s alright with you, this old geezer will be haulin’ himself back "
 "east, to an overdue retirement."
 msgstr ""
-"Pane, milorde, pane králi, jsem poctěn, opravdu jsem, ale chci jít domů. "
-"Jestli vám to nevadí, tenhle starý dědek se odveze zpátky na východ, do "
-"předčasného důchodu."
+"Pane, uh, mylorde, uh, pane Králi, jsem poctěn, opravdu jsem, ale chci jít "
+"domů. Jestli Vám to nevadí, tenhle starý veterán odkvačí zpátky na východ, "
```

Comment:
Já bych dal velké V v dopise, ne v přímé řeči.
```suggestion
"Pane, uh, mylorde, uh, pane králi, jsem poctěn, opravdu jsem, ale chci jít "
"domů. Jestli vám to nevadí, tenhle starý veterán odkvačí zpátky na východ, "
```

Reply:
Ano, z hlediska mluvčího to je sice přímá řeč, ale z hlediska hráče vlastně psané sdělení. Není to namluveno, hráč si to čte. Ale dejme tomu.